iib/workers/tasks/opm_operations.py

```diff
@@ -171,10 +178,14 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
             ret = rpc_proc.poll()
             # process has terminated
             if ret is not None:
-                stderr = rpc_proc.stderr.read()
-                if 'address already in use' in stderr:
+                if rpc_proc.stderr is None:
```

Comment:
Can you please explain why we added failure when stderr is not captured?

Reply:
Of course. So this part of the code will raise Exception one way or other. However there was this MYPY error `error: Item "None" of "Optional[IO[str]]" has no attribute "read"`. Basically I just added check for value, that is used later (and can be possibly empty). 
I can make this change in separated PR, however I believe this is not change of logic - either way exception will be thrown.

Reply:
Should be okay now.

Reply:
@chandwanitulsi stderr can be None, according to https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess.stderr . I am not sure if it ever happened or if it is possible, however in case that `ret` would be set and `stderr` would be `None`, there would not be relevant error message. Please check solution now (error message was changed).